README.md

```diff
@@ -54,7 +54,7 @@ being able to be run multiple times without needing user intervention.
 The descheduler pod is run as a critical pod to avoid being evicted by itself,
 or by the kubelet due to an eviction event. Since critical pods are created in the
 `kube-system` namespace, the descheduler job and its pod will also be created
-in `kube-system` namespace.
+in the `kube-system` namespace.
```

Comment:
nit: `in the same namespace` sounds a bit better since `kube-system` is mentioned only once

Reply:
Fixed